### ISSUE TITLE
Shortcut for timeUnit?

### ISSUE THREAD

**Issue by kanitw** [altair-viz/altair#849]
From reading the recent tutorial, I just notice that aggregation has a text shortcut e.g., `mean(field_name)`,  but timeUnit does not (e.g., `month(date)` does not work).  

So I wonder if this is intended?

**Comment by kanitw** [comment]
I also think that 

```
transform_timeunit(month='month(date)')
```

would be nicer than 

```
transform_timeunit(
    "month", field="date", timeUnit="month"
)
```

Here are a few reasons:
1) It's clear that month is the output.  (The current syntax doesn't make it very clear to new users that the first parameter is output as they do not know the convention yet.) 
 
2) It's a bit more consistent with how `transform_aggregate` assign the Vega-Lite "as" operator. 


That said, I'm not sure how to best make `transform_bin` consistent with these yet.

**Comment by jakevdp** [comment]
This has been available since version 2.0
